src/modules/discovery/hosts.py

```diff
@@ -38,44 +41,68 @@ def get_service_account_file(self, file):
         except IOError:
             pass
 
-class AzureMetadataApi(Vulnerability, Event):
-    """Access to the Azure Metadata API exposes information about the machines associated with the cluster"""
-    def __init__(self, cidr):
-        Vulnerability.__init__(self, Azure, "Azure Metadata Exposure", category=InformationDisclosure)
-        self.cidr = cidr
-        self.evidence = "cidr: {}".format(cidr)
-
 class HostScanEvent(Event):
-    def __init__(self, pod=False, active=False, predefined_hosts=list()):
-        self.active = active # flag to specify whether to get actual data from vulnerabilities
-        self.predefined_hosts = predefined_hosts
+    pass
 
-class HostDiscoveryHelpers:
-    @staticmethod
-    def get_cloud(host):
+# for comparing prefixes
+class InterfaceTypes(Enum):
+    LOCALHOST = "127"
+
+class HostDiscoveryUtils:
```

Comment:
I think Utils is more suited for these sort of functions. Helpers are generally small and not contain much logic, (like a helper function for converting string to int), The functions here are a bit more complex, they have conditions and generators.. That's why I prefer Utils instead of Helpers